frontend/package.json

```diff
@@ -45,7 +45,7 @@
     "prettier": "^3.3.2",
     "ts-loader": "^9.5.1",
     "typescript": "^5.3.3",
-    "webpack": "^5.92.1",
+    "webpack": "^5.94.0",
```

Comment:
Good that you updated package, but please try to separate commits. Each PR should contain code for same stuff. Please keep that in mind for future commits.